cmd/main.go

```diff
@@ -31,6 +31,10 @@ func main() {
 			&pluginv1.Plugin{},
 			&pluginv2.Plugin{},
 		),
+		cli.WithDefaultPlugins(
+			&pluginv1.Plugin{},
+			&pluginv2.Plugin{},
+		),
```

Comment:
Why we need to: 

```
		cli.WithPlugins(
			&pluginv1.Plugin{},
			&pluginv2.Plugin{},
		),
		cli.WithDefaultPlugins(
			&pluginv1.Plugin{},
			&pluginv2.Plugin{},
		),
```

What it the diff between plugin and default plugin? Is not it redundant?

Reply:
Imagine the case where there are multiple plugins per project version available to a user, ex. two different types of Go scaffolds that support project version 2. One of those two should be defaulted to in case the user doesn’t set `—plugins` on `init` or they don’t have a `layout` key in their config (project version 1). However the `CLI` won’t know which it should default to unless one is explicitly set. See https://github.com/kubernetes-sigs/kubebuilder/pull/1403#discussion_r388496659 for more details on why I chose to set defaults explicitly.